### PR TITLE
WOE 25 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/lord_skitters_butcher.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Lord Skitter's Butcher
+ManaCost:2 B
+Types:Creature Rat Peasant
+PT:2/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ When CARDNAME enters the battlefield, choose one —
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lord_skitters_butcher.txt:5]: use ABILITY

File: forge-gui/res/cardsfolder/upcoming/malevolent_witchkite.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Malevolent Witchkite
+ManaCost:4 B B
+Types:Creature Dragon Warlock
+PT:5/4
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, sacrifice any number of artifacts, enchantments, and/or tokens, then draw that many cards.
+SVar:TrigSac:DB$ Sacrifice | Defined$ You | Amount$ SacX | SacValid$ Enchantment,Artifact,Card.token | RememberSacrificed$ True | Optional$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ DrawX | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/malevolent_witchkite.txt:8]: missing Sub

File: forge-gui/res/cardsfolder/upcoming/moment_of_valor.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Moment of Valor
+ManaCost:2 W
+Types:Instant
+A:SP$ Charm | Choices$ DBIndestructible,DBDestroy
+SVar:DBIndestructible:DB$ Untap | ValidTgts$ Creature | SubAbility$ DBPump | SpellDescription$ Untap target creature. It gets +1/+0 and gains indestructible until end of turn.
+SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Indestructible
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/moment_of_valor.txt:6]: +1 missing

File: forge-gui/res/cardsfolder/upcoming/mosswood_dreadknight_dread_whispers.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
+Name:Mosswood Dreadknight
+ManaCost:1 G
+Types:Creature Human Knight
+PT:3/2
+K:Trample
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigEffect | TriggerDescription$ When CARDNAME dies, you may cast it from your graveyard as an Adventure until the end of your next turn.
+SVar:TrigEffect:DB$ Effect | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | Duration$ UntilTheEndOfYourNextTurn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mosswood_dreadknight_dread_whispers.txt:7]: ForgetOnMoved has wrong zone

File: forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Name:Picnic Ruiner
+ManaCost:1 R
+Types:Creature Goblin Rogue
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | IsPresent$ Creature.YouCtrl+powerGE4 | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks while you control a creature with power 4 or greater, CARDNAME gains double strike until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt:5]: > If you controlled a creature with power 4 or greater when you declared Picnic Ruiner as an attacker, it doesn't matter whether you still control one as its ability resolves.

File: forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+SVar:HasAttackEffect:TRUE
+AlternateMode:Adventure
+Oracle:Whenever Picnic Ruiner attacks while you control a creature with power 4 or greater, Picnic Ruiner gains double strike until end of turn.
+
+ALTERNATE
+
+Name:Stolen Goodies
+ManaCost:3 G
+Types:Sorcery Adventure
+A:SP$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select any number of target creatures you control to distribute counters to | CounterType$ P1P1 | CounterNum$ 3 | TargetMin$ 0 | TargetMax$ 4 | DividedAsYouChoose$ 4 | SpellDescription$ Distribute three +1/+1 counters among any number of target creatures you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt:17]: 4?

File: forge-gui/res/cardsfolder/upcoming/provisions_merchant.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Name:Provisions Merchant
+ManaCost:2 G G
+Types:Creature Beast Peasant
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_food_sac | TokenOwner$ You
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPumpAll | TriggerDescription$ Whenever CARDNAME attacks, you may sacrifice a Food. If you do,  If you do, attacking creatures get +1/+1 and gain trample until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/provisions_merchant.txt:7]: description typo

File: forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt (hunk 5 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt:5]: Does this need to be done with a `checksvar` ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/picnic_ruiner_stolen_goodies.txt:5]: that's no difference
you need NoResolvingCheck$

File: forge-gui/res/cardsfolder/upcoming/picklock_prankster_free_the_fae.txt [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+Oracle:Flying, vigilance
+
+
+ALTERNATE
+
+Name:Free the Fae
+ManaCost:1 U
+Types:Instant Adventure
+A:SP$ Mill | NumCards$ 4 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill four cards. Then put an instant, sorcery, or Faerie card from among the milled cards into your hand.
+SVar:DBChangeZone:DB$ ChangeZone | Hidden$ True | Origin$ Graveyard,Exile | Destination$ Hand | ChangeType$ Card.IsRemembered+Faerie,Card.IsRemembered+Sorcery,Card.IsRemembered+Instant | SelectPrompt$ Put an instant, sorcery, or Faerie card from among the milled cards into your hand | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/picklock_prankster_free_the_fae.txt:19]: Mandatory missing

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**